Runtimes/Core/core/CMakeLists.txt

```diff
@@ -265,6 +265,12 @@ if(SwiftCore_ENABLE_VECTOR_TYPES)
     "${CMAKE_CURRENT_BINARY_DIR}/SIMDVectorTypes.swift")
 endif()
 
+if(APPLE AND BUILD_SHARED_LIBS)
+  target_link_options(swiftCore PRIVATE
+    "SHELL:-Xlinker -headerpad_max_install_names"
+  )
```

Comment:
To be consistent with the rest of this file, can we keep the closing `)` on the previous line? Thanks